services/brig/src/Brig/Team/API.hs

```diff
@@ -156,6 +157,17 @@ routesPublic = do
     Doc.response 404 "No pending invitations exists." Doc.end
     Doc.response 409 "Multiple conflicting invitations to different teams exists." Doc.end
 
+  get "/teams/:tid/size" (continue teamSizePublicH) $
+    accept "application" "json"
+      .&. header "Z-User"
+      .&. capture "tid"
```

Comment:
Can you please add a note about this endpoint returning potentially stale size as it depends on `refresh_interval` of the index which is in seconds (5 seconds in prod, iirc).